index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;0,500;0,700;1,100;1,400;1,700&display=swap"
+    rel="stylesheet">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <div class="wrapper">
+    <header>
+      <nav>
+        <a href="#">
+          <div class="logo">
+            My Portfolio
+          </div>
+        </a>
+        <div class="menu">
+          <a href="#">
+            <img src="src/icons/hamburger.svg" alt="">
+          </a>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <h2 class="main-title">I'm Sasan<br>Glad to see you!</h2>
+      <p class="main-description">
+        I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <p class="connect">LET'S CONNECT</p>
+      <div class="connect-icons">
```

Comment:
* [ ]  kindly update your code to make it more semantic and for the best HTML practices : the social icons should be listed inside a list `<ul> & <li>` instead of normal divs, and each social icon can be left as an `<a>` that contains the link for that icon

   example:
> 
> ```
> <ul class='...'>
> <li class='...'><a href='...' ><svg>...</svg></a></li>
> <li class='...'><a href='...' ><svg>...</svg></a></li>
> ...
> <li class='...'><a href='...' ><svg>...</svg></a></li>
> </ul>
> ```